contracts/test/script/compare-storage-layout.js

```diff
@@ -0,0 +1,68 @@
+const { execSync } = require("child_process");
```

Comment:
It's a bit difficult to follow this script or be certain that it would always fail if there we made a mistake.

Are there now existing solution for checking?

I found for example https://github.com/Rubilmax/foundry-storage-check but that doesn't seem to be geared towards using a library and isn't published to npm.

Reply:
true, i'll look for command line tools to help to see if that simplifies a lot more

Reply:
to no avail with making it work with command line but i simplified the way the type field was cleaned and also compared to a contract that definitely does not have the same storage layout to prove that the script works as expected https://github.com/EspressoSystems/espresso-network/commit/6f1802f6a750d79000685ecaff9edfe8a3c88ea2

and also compared with "invalid" staketable contracts that either have a missing field or the fields reordered https://github.com/EspressoSystems/espresso-network/commit/8356f22661c6f4b7281dcd7e3e9c97b70ab1e824